### DISCUSSION TITLE
Question about Boiler Activation threshold and multiple radiators in one room

### DISCUSSION THREAD

**Discussion by TTLucian** [jmcollin78/versatile_thermostat#758] [Q&A]
Hello! First I would like to thank JMC for this integration! It's awsome and works great with my current setup. 🙏 👏 
I have a question about "Boiler Activation threshold" parameter in the following situation. I use Sonoff TRVZBs on all the radiators with a central boiler configuration. Some rooms have 1 radiator, others have 2 and others have 3. I've configured vtherms for each room, each having 1, 2 or 3 TRVs as underlyings. I want to turn on the boiler only if at least 2 rooms are requesting heating at the same time. How can I can't reliably achieve this? Because seting this parameter at 2 devices can cause only one room with 2 radiators to turn on the boiler, same problem if setting to 3 for the room with 3 radiators.

**Comment by jmcollin78** [comment]
Hello @TTLucian ,

There is no way to do that simply out of the box. This need seems strange for me. The boiler should be activated if some TRV needs it indeependantly of the localization of the TRV. What is your use case ?

**Reply by tbrasser** [reply]
I think this is not really to prevent shortcycling, but just to save energy: "If only my "unimportant room" demand heating, but not rest of the housle, don't turn on boiler"


However IMO, this can better be handled by not activating any demand in the unimportant room in the first place, since many trv's will go in error mode after a while when valve is open but no temperature increase detected (valve error).

This will also allow for real demand-based controll of the (modulating) boiler (or pwm on on/off boiler), without any "weird" exceptions.

**Comment by doubledrat** [comment]
Could you use one vtherm per radiator and only give one of them per room control over the boiler?

**Reply by jmcollin78** [reply]
yes you can do that. You choose which vtherm controls the heater. So it is possible.
